src/status_im/ui/screens/contacts/subs.cljs

```diff
@@ -98,10 +99,14 @@
     (->> (remove :pending? (vals groups))
          (sort-by :order >))))
 
-(reg-sub :contact
+(reg-sub :current-contact-identity
   (fn [db]
-    (let [identity (:contacts/identity db)]
-      (get-in db [:contacts/contacts identity]))))
+    (:contacts/identity db)))
+
+(reg-sub :contact
+   (fn [db]
```

Comment:
why don't you use here `:current-contact-identity` ?

Reply:
so it's better to use `current-contact-identity` and `contacts` subscriptions

Reply:
```
(reg-sub :contact
  :<- [:get-contacts]
  :<- [:current-contact-identity]
  (fn [[contacts identity]]
    (get contacts identity)
```

Reply:
in this case this subscription will be recalculated only when  contacts or identity will be changed but not app-db

Reply:
nice, thanks!